isofit/utils/6S_to_sRTMnet.py

```diff
@@ -68,7 +68,3 @@ def cli(sixs, emulator, aux, output, predicts):
     predict(emulator, sixs, resample, scaler, predicts)
 
     print("Done")
-
-
-if __name__ == "__main__":
-    cli()
```

Comment:
I believe we want to keep this one for this script. This is just a utils script that may get removed and is not presently a part of the main CLI, so without this there's no way to call it.

Reply:
@jammont If that is the case, should it even live within the `isofit` Python library? Is the `scripts/` directory maybe a better home? Is the script just for developer use?

Reply:
When we first wrote it that was a question brought up as well, where should it live. Now that we have the `scripts/` directory, it very well could be moved there. And yes, just for developer use. We've also considered opening another repo like `isofit-research` to house things like this.

@pgbrodrick, would you like to have an opinion on this?

Reply:
There's a broader conversation to be had about where non-bulk-workflow utility scripts (e.g. this one or downloads.py) should live.  I don't see that this needs hold things up...can't we just another line into __main__.py to facilitate it's call?

Reply:
Done.